packages/server/src/arpa_reporter/services/persist-upload.js

```diff
@@ -244,7 +244,15 @@ async function bufferForUpload(upload) {
 async function jsonForUpload(upload) {
     return tracer.trace(
         'jsonForUpload',
-        async () => Cryo.parse(await fs.readFile(jsonFSName(upload), { encoding: 'utf-8' })),
+        async () => {
+            const file = await tracer.trace('fs.readFile', async (span) => {
+                const f = await fs.readFile(jsonFSName(upload), { encoding: 'utf-8' });
+                const { size } = await fs.stat(jsonFSName(upload));
+                span.setTag('filesize-kB', Math.round(size / (2 ** 10)));
```

Comment:
If we are adding some tags on here, could we add the tenant id / reporting period id as well? Thoughts?

Reply:
We can add those!  Do you want them on this span specifically, or somewhere else?

Reply:
I think this span makes sense. We can always start with this and see if we want to add it elsewhere.

Reply:
I pushed another change and it looks like I succeeded in adding these two tags!

<img width="520" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/11449340/994ebee8-2314-4684-b28e-2e06a0026ec5">

https://app.datadoghq.com/apm/trace/7692138087698454584?colorBy=service&env=sandbox&graphType=flamegraph&shouldShowLegend=true&sort=time&spanID=7692138087698454584&spanViewType=metadata&timeHint=1693189604002.007